src/main/scala/com/fulcrumgenomics/alignment/NeedlemanWunschAligner.scala

```diff
@@ -25,21 +25,36 @@
 package com.fulcrumgenomics.alignment
 
 import com.fulcrumgenomics.FgBioDef._
+import com.fulcrumgenomics.alignment.NeedlemanWunschAligner.Mode.{Global, Glocal}
 import com.fulcrumgenomics.alignment.NeedlemanWunschAligner._
+import enumeratum.EnumEntry
 import htsjdk.samtools.CigarOperator
 
-import scala.collection.mutable
+import scala.collection.{immutable, mutable}
 import scala.math.max
 
 object NeedlemanWunschAligner {
+
+  sealed trait Mode extends EnumEntry
+
+  /** Enum to represent alignment modes supported by the NeedlenameWunschAligner */
+  object Mode extends FgBioEnum[Mode] {
+    /** Alignment mode for global pairwise alignment. */
+    case object Global extends Mode
+    /** Alignment mode for global alignment of query sequence to local region of target sequence. */
+    case object Glocal extends Mode
+
+    override def values: immutable.IndexedSeq[Mode] = findValues
+  }
```

Comment:
Did you test this on the command line (i.e. specify the value for the mode)?  Last time I had an EnumEntry inside the object, it didn't work, so I needed to move it out of the object.  This only affects when the `Mode` type is used as the type for a command line argument.

Reply:
I didn't.  I almost made this a straight up scala Enumeration, because I don't see it being something that's command line exposed (i.e. in most applications you'll want one mode of the other, not a user-choice).  I'll move it to the top-level though just in case.